posixlib/src/main/scala/scala/scalanative/posix/getopt.scala

```diff
@@ -3,6 +3,9 @@ package posix
 
 import scalanative.unsafe._
 
+@deprecated(
+  "getopt is no longer part of POSIX 2018 and will be moved to unistd"
```

Comment:
```suggestion
  "getopt is no longer part of POSIX 2018 and will be removed. Use unistd instead"
```

Ah, I found getopt-related stuffs already in unistd. `will be moved to unistd` is not correct.